process/testing/winfv/create_kubeadm_cluster.sh

```diff
@@ -82,12 +82,12 @@ echo "Calico is running."
 kubectl taint nodes --all node-role.kubernetes.io/master-
 
 # strict affinity
-curl -O -L  https://github.com/projectcalico/calicoctl/releases/download/v3.17.1/calicoctl
+curl -L https://github.com/projectcalico/calico/releases/download/v3.27.0/calicoctl-linux-amd64 -o calicoctl
```

Comment:
```suggestion
curl -sSf -L --retry 5 https://github.com/projectcalico/calico/releases/download/v3.27.0/calicoctl-linux-amd64 -o calicoctl
```
(to try to mitigate possible transient server issues)

Reply:
Updated. Thanks for the suggestion.